int32 object_id
float64 distance_m
int32 risk_level
geometry_msgs/Point relative_position
geometry_msgs/Point global_position
